item/array.go

```diff
@@ -0,0 +1,54 @@
+package item
+
+import (
+	"fmt"
+	"github.com/zimnx/YamlSchemaToGoStruct/set"
+)
+
+// Array is an implementation of Item interface
+type Array struct {
+	item Item
+}
+
+// Type implementation
+func (item *Array) Type(suffix string) string {
```

Comment:
Could you please use a different name for the variable than 'item' as it is the package name?